modules/perception/onboard/dag_streaming.cc

```diff
@@ -230,7 +230,7 @@ void DAGStreamingMonitor::Run() {
   }
 }
 
-Subnode* DAGStreaming::GetSubnodeByName(std::string name) {
+Subnode* DAGStreaming::GetSubnodeByName(const std::string &name) {
```

Comment:
Thanks a lot for the contribution. The header file need be updated too to match the new change. 

Thanks.